docs/qdmi/qiskit_backend.md

```
@@ -0,0 +1,176 @@
+## Installation
+
+Install MQT Core with Qiskit support:
+
+::::{tab-set}
+:sync-group: installer
+
+:::{tab-item} {code}`uv` _(recommended)_
+:sync: uv
+
+```console
+$ uv pip install "mqt-core[qiskit]"
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)